alkis/.htaccess

```diff
@@ -0,0 +1,27 @@
+Options +FollowSymLinks
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+# AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+# AddType text/turtle .ttl
+
+RewriteEngine on
+
+#Rewrite rules for ALKIS vocabulary extension
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml|text/\*|\*/\*)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^alkis$ https://github.com/tubav/alkis/blob/master/  [R=303,NE,L]
```

Comment:
Rules are relative to current directory, so `^alkis$` would make the end point https://w3id.org/alkis/alkis.  You probably just want `^$`.  Same below.

Reply:
Also done :)